spec/requests/get_services_from_user_spec.rb

```diff
@@ -0,0 +1,64 @@
+RSpec.describe 'GET /services/user/:user_id' do
```

Comment:
maybe a bit picky, and this is how it's defined in the docs, but maybe we should have `/services/users/:user_id`? Plural on the `users` to keep it consistent with `versions` and `services`.

Reply:
```suggestion
RSpec.describe 'GET /services/users/:user_id' do
```